src/dashboard/src/components/administration/forms.py

```diff
@@ -64,6 +64,16 @@ class StorageSettingsForm(SettingsForm):
         label="Full URL of the storage service")
 
 
+class ChecksumSettingsForm(SettingsForm):
+    CHOICES = (
+        ('md5', 'MD5'),
```

Comment:
It was requested (I confirmed with Justin). We don't think the security is a huge issue in this case since we're just using it as a checksum to detect corruption in files.